packages/cryptoplease_link/lib/src/swap/create_swap_fee_payment.dart

```diff
@@ -0,0 +1,53 @@
+import 'package:cryptoplease_link/src/constants.dart';
+import 'package:solana/encoder.dart';
+import 'package:solana/solana.dart';
+
+Future<Message> createSwapFeePaymentMessage({
+  required SolanaClient client,
+  required Ed25519HDPublicKey aReceiver,
+  required Ed25519HDPublicKey aSender,
+  required Commitment commitment,
+  required int amount,
+}) async {
+  // Swap fee is always paid in mainnet USDC
+  final mint = mainnetUsdc;
+
+  final shouldCreateAta = !await client.hasAssociatedTokenAccount(
```

Comment:
It's always platform, and it should always have ATA, so it doesn't make sense to check it here.